Documentation/project-docs/benchmarking.md

```diff
@@ -52,15 +52,24 @@ If you want to run your benchmarks without spawning a new process per benchmark
 
 1. Before you start benchmarking the code you need to build entire CoreFX in Release which is going to generate the right CoreRun bits for you:
 
-        C:\Projects\corefx> build.cmd -release -buildArch=x64
+        C:\Projects\corefx>build.cmd -release -buildArch=x64
 
 After that, you should be able to find `CoreRun.exe` in a location similar to:
 
         C:\Projects\corefx\bin\testhost\netcoreapp-Windows_NT-Release-x64\shared\Microsoft.NETCore.App\9.9.9\CoreRun.exe
 
 2. Create a new .NET Core console app using your favorite IDE
 3. Install BenchmarkDotNet (0.11.1+)
-4. Define the benchmarks
+4. Define the benchmarks and setup runner
+
+```cs
+...
```

Comment:
I think that the `...` could be removed so only "```" would stay

Reply:
done